src/main/proto/wfa/measurement/protocol/liquid_legions_sketch_aggregation_v2.proto

```diff
@@ -99,9 +111,24 @@ message LiquidLegionsSketchAggregationV2 {
     org.wfa.measurement.config.LiquidLegionsV2SetupConfig.RoleInComputation
         role = 1;
 
-    // The list of ordered duchy ids in the MPC ring, with the first element
-    // being the first non-aggregator and the last element being the aggregator.
-    repeated string duchy_order = 2;
+    message DuchyPublicKey {
+      // Name of the duchy who owns the key.
+      string duchy_id = 1;
+      // The ElGamal public key.
+      wfa.common.ElGamalPublicKey public_key = 2;
+    }
+    // The list of duchy public keys used in this computation.
+    // The list is sorted by the duchy order in the MPC ring, with the first
+    // element being the first non-aggregator and the last element being the
+    // aggregator.
+    repeated DuchyPublicKey duchy_keys = 2;
+
+    // list of local requisitions that are required for this computation.
+    repeated RequisitionKey keys = 3;
+
+    // Total number of MetricRequisitions across all duchies used in this
+    // computation.
+    int32 total_requisition_count = 4;
```

Comment:
Would this be equal to number of EDPs who participate in this requisition?

Reply:
Yep